draft-rfcs/working-group-terminology.md

```diff
@@ -2,96 +2,274 @@
+groups and having a period of time soliciting applications from the community,
+and then approving a subset of those applications.
 
-- Community groups are not required to follow any of the guidelines mentioned above.
-- As such community groups are not officially endorsed by the Rust project.
+> **DRAFT NOTE** Should this application come in the form of an RFC? My
+> inclination is yes, however it could just create more churn and drama than
+> needed. Posting in a thread on internals as was done previously might
+> be enough.
 
-# Drawbacks
+#### Application Checklist
+
+This not meant to be formal list of questions to be answer, however the
+application should try to address these points.
+
+- What value do you see your group bringing to the organisation?
+- Why should the project your group?
+- What are the goals of your group?
+- What are explicitly non-goals of your group?
+
+These first four points should also serve as the basis for the charter for the
+working group.
 
-[drawbacks]: #drawbacks
+- Is your group long-running or temporary?
+  - If it is temporary, how long do you see it running for?
+- What is the long-term vision of your group?
+- What do you expect the relationship to the organisation be?
+- How do you want to establish accountability?
+- If applicable, which other working groups or teams do you expect to have close
+  contact with?
+- What are the short-term goals of your group?
+- Who are the initial shepherds/leaders? (This is preferably 2–3 individuals,
+  but not required.)
+- How do you intend to make your work accessible to people outside your group?
+- Where do you see your group needing help?
+- What is your preferred way for the core team to contact you?
+
+### Working Group Evaluation
+
+Working group shepherds should be regularly participating in the project leads
+discussion channels so that the rest of the organisation is aware of the work
+and can help where needed.
+
+If a working group has demonstrated consistent productivity over a significant
+period time, and there is consensus that there is significant future work, it
+may become a Rust team. Conversely if there is consensus that the work is
+"complete" to the point that there's there is little benefit to continuing the
+working group, it may be wound down.
+
+The wind down process of a working group involves communicating the wind down to
+the community and the archival or transfer of ownership of the relevant projects.
+
+## Project Groups
+
+A Project Group is a group of people working on a particular project or
+responsibilities at the behest of an official Rust team. Examples of this would
+include [FFI Unwind], [Inline ASM], and [Safe Transmute].
+
+The goal of a project is build a community or formalise and existing community
+around a particular feature or project in the in the organisation, and use this
+space to discuss and iterate on that feature.
+
+Part of building a community is removing some of the institutional memory that
```

Comment:
Not removing, perhaps rather stewarding? Providing a home for this discussion rather than having it end up buried in the RFC thread?